cypress/e2e/suggestion.cy.ts

```diff
@@ -0,0 +1,29 @@
+const suggestionId = Cypress.env("CYPRESS_SUGGESTION_ID")
+
+describe("suggestion detail page", () => {
```

Comment:
@rensdoodeman the main functionality of this page is entering what instrument you plan on playing, yet there seems to be missing any tests for this. Are you considering writing those tests?